docs/spelling_wordlist

```diff
@@ -979,3 +979,5 @@ xxxxx
 yesno
 Zealand
 Zope
+includeSubDomains
```

Comment:
These terms should be surround by doubebackticks instead of added to the word list.